datatypes/src/commonMain/kotlin/at/asitplus/crypto/ecmath/ECMath.kt

```diff
@@ -189,3 +190,38 @@ inline fun ECPoint.times(v: Long) = v * this
 
 @Suppress("NOTHING_TO_INLINE")
 inline fun ECPoint.times(v: ULong) = v * this
+
+/** computes uG+vQ */
```

Comment:
more efficient calculation of uG+vQ by caching G+Q; the operation is needed in ECDSA

Reply:
I'm particularly bad at maths (way worse than one would assume, given my level of education), so I'm just gonna trust you on that one